index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio Homepage</title>
+</head>
+
+<body>
+  <div id="toolbar">
+    <div id="logoPlaceholder">
+      <p id="logoText">Welcome</p>
+    </div>
+    <input id="headerButton" type="image" src="Images/Enabled.png" alt="Menu">
+  </div>

```

Comment:
Kindly make use of the `header` element instead of a `div` for introductory content for your page, in your case your toolbar. You can read more about why it is important to use a header tag [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).